torch/fx/node.py

```diff
@@ -915,10 +916,10 @@ def map_aggregate(a: Argument, fn: Callable[[Argument], Argument]) -> Argument:
             dict.__setitem__(rv, k, map_aggregate(v, fn))
         return rv
     elif isinstance(a, slice):
-        return slice(
+        return slice(  # type: ignore[return-value]
```

Comment:
Will this work, will we need to do typing.overrload to properly handle it?

Reply:
Okay, Slice is now generic, but may require a typing_extensions / typeshed / mypy update: https://github.com/python/typeshed/issues/8647

Reply:
https://github.com/python/typeshed/pull/11637

Reply:
@Skylion007  Well, I'm _not_ entirely sure this will work, but it seems to represent what is actually going on!, gives better typing at point of use, and running some tests locally seemed to work FWIW.

(This actually emerged from tweaking some typing elsewhere...)

I'm seeing some tests fail due to backwards compatibility checks which need human approval to tweak. I'll see how the whole thing shakes out before I actually contact a human and ask for lenience... 😬 

---

As for generic slice - well, it seems very hard to deduce what type is actually being sliced on line 919 there.

Reply:
Actually was referring to the typing of `Argument`, there was a comment that the slice in that TypeAlias should be typed, but it wasn't because of a lack of support. That should be fixed now. :)

Reply:
Link is here: https://github.com/pytorch/pytorch/blob/f97307f46312670ba6995822046ba03ad6a1eadc/torch/fx/node.py#L48